modules/ROOT/pages/https-reference.adoc

```diff
@@ -2,9 +2,33 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:keywords: configure, HTTPS endpoint
+
+You can automatically deploy an API proxy using an HTTP listener (inbound HTTP endpoint) with minimal setup. You can use this approach if you do not foresee any security concerns in your Anypoint Platform API setup. Otherwise, you must configure your API proxy with an HTTPs listener (inbound HTTPS endpoint), instead of an HTTP listener.
+
+MuleSoft requires that you provide your own certificate when configuring your HTTPs setup. The certificate you use can be a self-signed certificate to be used for demo purposes, custom TLS certificate with validity within your organization, or a trusted Certificate Authority (CA) certificate.
+
+If your backend uses HTTPs with a trusted CA certificate, you do not need to perform any special action to communicate with it from the API proxy. If your backend uses HTTPs with a self-signed certificate or custom TLS certificate, you must configure the API proxy requester with the corresponding certificate.
+
+Depending on your business use case, you can use any of the following recommended ways to configure your API proxy to use a certificate when setting up an HTTPS listener:
```

Comment:
I think we can omit this in favor of the previous sentence, or tweaking the previous sentence a bit.
```suggestion

```

Reply:
Done! Thanks!